File: Day 13/tipCalculator.c
#include <stdio.h>
#include <math.h>

int main()
{

    int plates;
    printf("Type the number of different plates: \n");
    scanf("%d", &plates);

    int prices[plates];
    int n[plates];

    int i;
    for(i=0; i<plates; i++){

        printf("Type the price of plate %d \n", i+1);
        scanf("%d", &prices[i]);

        printf("Type the unities of plate %d \n", i+1);
        scanf("%d", &n[i]);

    }

    int j;
    int totalPrice = 0;
    int tip;
    for(j=0; j<plates;j++){
        totalPrice += prices[j]*n[j];
    }
    tip = totalPrice*0.1;

    printf("The total price is %d \n", totalPrice);
    printf("The tip is %d! \n", tip);


    return 0;
}

File: Day 13/tipCalculator.js

function plate(price, n){
    this.price = price;
    this.n = n;
}

function constructMenu(){
    
    let nPlates = prompt("How many different plates?")
    
    let plates = []
    
    let totalPrice = 0

    const TIP = 0.1

    for(let i=0; i<nPlates; i++){
    
        plates.push( new plate( prompt(`Type the price of the plate ${i+1}:`), prompt("Type the unities consumed: ") ) )
           
    }

    console.log(plates)

    for(let i=0; i < plates.length; i++){

        totalPrice += plates[i].price * plates[i].n 

    }

    tip = totalPrice*TIP

    console.log(`The total price is ${totalPrice}`)
    console.log(`The tip for you is ${tip}!`)

}

constructMenu()
